### PR TITLE
[stable/atlantis] Added support for existing GCP Service Account Secret

### PR DESCRIPTION
**Pull request by jonnydford**
#### What this PR does / why we need it:
Adds support for existing GPC Service Account to use with Atlantis, set `GOOGLE_APPLICATION_CREDENTIALS` to file path of this account. 

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/atlantis/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "v0.7.1"
 description: A Helm chart for Atlantis https://www.runatlantis.io
 name: atlantis
-version: 3.5.3
+version: 3.5.4
```
Comment (lkysow) [inline stable/atlantis/Chart.yaml]:
```suggestion
version: 3.6.0
```
Since it's new functionality it should be a minor release

**Review by lkysow — CHANGES_REQUESTED** [review]

**Comment by lkysow** [comment]
@jonnydford I think you need to update the chart.yaml because another PR bumped the version

**Comment by maorfr** [comment]
need to signoff that commit

**Pull request closed without merging**